### DISCUSSION TITLE
Trying to use this with my own custom movement component.

### DISCUSSION THREAD

**Discussion by salvinoto** [Sixze/ALS-Refactored#165] [Q&A]
So I have my own movement component for my FPS game, and I want to use this as my 3rd person mesh that other players can see. How can I use this in my own character blueprint for the animations without reparenting the character.

**Comment by nekwo** [comment]
you would have to merge the movement components since this uses a custom ALS movement component and then you would have to retarget. keep in mind you can merge the movement components and still reparent as long as the bone names are the same
